### PR TITLE
Break down session length per app

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
@hodgestar @imsickofmaps @justinvdm ready for review

File: test/personal.test.js [hunk 1]
```diff
@@ -116,8 +116,8 @@ describe("app", function() {
             it('should publish metrics', function () {
                 return tester
                     .setup(function(api) {
-                        api.kv.store['session_length_helper.foodacom.sentinel'] = '2000-12-12';
-                        api.kv.store['session_length_helper.foodacom'] = 42;
+                        api.kv.store['session_length_helper.' + api.config.store.name + '.foodacom.sentinel'] = '2000-12-12';
+                        api.kv.store['session_length_helper.' + api.config.store.name + '.foodacom'] = 42;
                     })
```
Comment (justinvdm) [inline test/personal.test.js:121]: I'm probably forgetting something, but I'm surprised this works. I think you would need to do `api.config.store.config.name` or `api.config.app.name` here (`api.config.app` is just a shortcut to `api.config.store.config.name`).  `api.config.store` contains all the configs (so for example, translation configs or reports configs would go on the object too).

Comment (justinvdm) [inline test/personal.test.js:121]: We also set the name to `'personal'` [above](https://github.com/praekelt/go-ndoh/blob/feature/issue-136-breakdown-session-length-stats-per-service/test/personal.test.js#L51), so maybe we could just put that here instead?

Comment (smn) [inline test/personal.test.js:121]: You're right "session_length_helper.undefined.foodacom.sentinel" is what's generated.

**Comment by justinvdm** [comment]
Other than the minor comment, :+1:

File: test/chw.test.js [hunk 2]
```diff
@@ -156,18 +159,18 @@ describe("app", function() {
...
-                          kv_store['session_length_helper.foodacom.sentinel'], '2014-04-04');
+                          kv_store['session_length_helper.' + im.config.name + '.foodacom.sentinel'], '2014-04-04');
 
                         var m_store = api.metrics.stores.test_metric_store;
                         assert.equal(
-                          m_store['session_length_helper.foodacom'].agg, 'max');
+                          m_store['session_length_helper.' + im.config.name + '.foodacom'].agg, 'max');
                         assert.equal(
-                          m_store['session_length_helper.foodacom'].values[0], 60);
+                          m_store['session_length_helper.' + im.config.name + '.foodacom'].values[0], 60);
```
Comment (justinvdm) [inline test/chw.test.js:173]: It might make sense to use `api.config.app` here too, but its not really important.

**Pull request merged**